client/trigger_dockerfile_lint.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/python
+
+import beanstalkc
+import json
+import os
+import sys
+
+if __name__ == "__main__":
+    name_space = sys.argv[1]
+    git_path = sys.argv[2]
+    filename = sys.argv[3]
+    notify_email = sys.argv[4]
+
+    # having '/' in a value used in os.path.join generates unexpected paths
+    if git_path[0] == "/":
+        git_path = git_path[1:]
+
+    if filename[0] == "/":
+        filename = filename[1:]
+
+    lint_job_data = {}
+
+    with open(os.path.join(os.environ['DOCKERFILE_DIR'], git_path, filename)) \
+    as f:
+        dockerfile = f.read()
```

Comment:
I can imagine one issue which is that environment variable isn't set. But we're explicitly setting that [here](https://github.com/CentOS/container-pipeline-service/pull/82/files/2819e64342fa9fe9ab6fce9d8e054a60ded1cdc7#diff-4b6033ad5d043bfedd9cae16205e449cR15).